shopinvader_reset_password/__openerp__.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Akretion (http://www.akretion.com).
+# @author Sébastien BEAU <sebastien.beau@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Shopinvader Reset Password",
+    "summary": "Give the possibility to send a email to reset the"
+               "password from odoo",
```

Comment:
@sebastienbeau Sources seems to target 8.0... WIP?

Reply:
WIP, I have done the PR for the extraction but not yet the migration